src/value.cc

```diff
@@ -862,7 +862,7 @@ ValuePtr::ValuePtr(double v)
 
 ValuePtr::ValuePtr(const std::string &v)
 {
-	this->reset(new Value(v));
+	this->reset(new Value(str_utf8_wrapper(v)));
```

Comment:
I think we should create the string wrapper in the `Value` constructor, not in the `ValuePtr` constructor, just to be clean; `ValuePtr` is really just a smart pointer wrapper around `Value`